src/landing/Landing.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react'
+import styles from './Landing.module.css'
+
+export default class Landing extends React.Component {
+  render() {
+    return (
+      <div>
+        <center>
```

Comment:
Avoid using obsolete HTML tags like <center>. For now, this is fine but should be changed in the future